setup.py

```diff
@@ -38,7 +38,7 @@
       zip_safe=False,
       test_suite='nose.collector',
       tests_require=['nose'],
-      install_requires=['Cython==0.21'],
+      install_requires=['Cython>=0.19'],
```

Comment:
given that OpenMS requires cython `> 0.21`, this is really helpful. right now, we install autowrap, which installs the older Cython (potentially uninstalling the newer Cython) and then we have to manually install the new Cython again.